Configuration/TCA/Module/Options.php

```diff
@@ -85,6 +85,12 @@
                 'type' => 'check',
             ],
         ],
+        'auto_orient_custom' => [
+            'label' => 'Use custom auto_orient (flop rotate 6 and 8 handling, based on image exif)',
```

Comment:
=> Use a `LLL:EXT` (XLIFF) label

Furthermore, I'd suggest to add a `description` field with some further details, such as a (textual) link explaining the problematic with some pieces of software.

Reply:
Yes, can be done, i leave that up to you :-)